ipaserver/install/installutils.py

```diff
@@ -164,7 +164,7 @@ def verify_fqdn(host_name, no_host_dns=False, local_hostname=True):
                          host_name)
             ex_name = socket.gethostbyaddr(host_name)
             logger.debug('Primary hostname for localhost: %s', ex_name[0])
-            if host_name != ex_name[0]:
+            if host_name not in ex_name and [host_name] not in ex_name:
```

Comment:
This is exactly the problem. Thanks for reaching out, but if I change to your example, it will raise the same error if the `/etc/hosts` was like `server aliasserver server.ipadomain aliasserver.ipadomain` using **is*" to check if is in tuple would validate dispit the index of FQDN. I think I tried that, reproduce this is very easy:-)

Reply:
please see the last comments, thanks!